src/entities/nestedExit/getQueryCallsAttributes.ts

```diff
@@ -0,0 +1,201 @@
+import { Token } from '../token';
+import {
+    NestedProportionalExitInput,
+    NestedSingleTokenExitInput,
+    NestedExitCallAttributes,
+} from './types';
+import { NestedPool, NestedPoolState } from '../types';
+import { TokenAmount } from '../tokenAmount';
+import { Address } from '../../types';
+
+export const getQueryCallsAttributes = (
+    input: NestedProportionalExitInput | NestedSingleTokenExitInput,
+    { pools }: NestedPoolState,
```

Comment:
Can't we just change this to be: `pools: NestedPool[]`?

Reply:
Yeap 👍